Resources/Prototypes/_NF/Entities/World/Debris/asteroids.yml

```diff
@@ -0,0 +1,1078 @@
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      radius: 28
+      floorPlacements: 108
+
+- type: entity
+  id: AsteroidSalvageNFSandSmall
+  parent: BaseAsteroidDebrisNFSand
+  name: Salvage Sand Asteroid Small
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 15
+      floorPlacements: 100
+
+- type: entity
+  id: AsteroidSalvageNFSandMedium
+  parent: BaseAsteroidDebrisNFSand
+  name: Salvage Sand Asteroid Medium
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 17
+      floorPlacements: 150
+
+- type: entity
+  id: AsteroidSalvageNFSandLarge
+  parent: BaseAsteroidDebrisNFSand
+  name: Salvage Sand Asteroid Large
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 20
+      floorPlacements: 200
+
+- type: entity
+  id: AsteroidSalvageNFSandHuge
+  parent: BaseAsteroidDebrisNFSand
+  name: Salvage Sand Asteroid Huge
+  noSpawn: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      blobDrawProb: 0.66
+      radius: 23
+      floorPlacements: 250
+
+# Chromite
+
+- type: entity
+  id: BaseAsteroidDebrisNFChromite
+  parent: BaseDebrisNF
+  name: Salvage Chromite Asteroid
+  abstract: true
+  components:
+    - type: MapGrid
+    - type: BlobFloorPlanBuilder
+      floorTileset:
+      - FloorChromite
+      blobDrawProb: 0.5
+      radius: 6
+      floorPlacements: 16
+    - type: SimpleFloorPlanPopulator
+      entries:
+        FloorChromite:
+          - id: RandomArtifactSpawner20
+            prob: 0.01
+            orGroup: rock
+          - id: ChromiteElementalSpawner
+            prob: 0.08
+            orGroup: rock
+          - id: WallRockChromite
+            prob: 0.65
+            orGroup: rock
+          - id: WallRockChromiteUranium
+            prob: 0.1
+            orGroup: rock
+          - id: WallRockChromitePlasma
+            prob: 0.05
+            orGroup: rock
+          - id: WallRockChromiteTin
+            prob: 0.05
+            orGroup: rock
+          - id: WallRockChromiteArtifactFragment
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockChromiteBananium
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockChromiteGold
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockChromiteQuartz
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockChromiteSalt
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockChromiteCoal
+            prob: 0.01
+            orGroup: rock
+          - id: WallRockChromiteSilver
+            prob: 0.01
+            orGroup: rock
+          - id: WallChromiteCobblebrick
+            prob: 0.1
+            orGroup: rock
+    - type: IFF
+      flags: HideLabel
+      color: "#7d75bf"
```

Comment:
```suggestion
      color: "#8178cc"
```

Chromite asteroids show up as a bit dark on the radar.  This makes them slightly easier to read.  In-game:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/8448dfed-0fe7-40e7-96d8-5450715f9b4e)
Should better stand out against the background and be readily distinguishable from wrecks.